ci/release/update-version.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
This is a file we use to update all the versioning numbers automatically when we do a release

Please update the sed commands to match and replace all instances where version numbers in the repo need updating